intro_calculating_cred_grain.md

```diff
@@ -0,0 +1,162 @@
+
+#### Weights:
+
+Because these plugins incorporate data from all sorts of contributions and output a single weighted, directed graph, there needs to be a way to quantitatively compare actions (e.g. authoring a GitHub issue vs responding or referencing it). To accomplish this, there’s a set of constants that determine weights for nodes and edges. For an example set of constants, see the below figure, captured from [sourcecred.io/cred/timeline/@sourcecred/](https://sourcecred.io/cred/timeline/@sourcecred/) (follow the link and click the “Show weight configuration” button).
+
+![](https://lh6.googleusercontent.com/BDSrcvjcCXXfKQ0NSiHIYuAh8fvTaE0Akl0g2-PLdB88fv5TXZS7btltL0YTe77TLOT7nyImct7sPrKATbAQKUwukM3bdxgVfIJCf-xDZo0nvGnFINMoqXq41glKAI9hA2EnThed)
+
+> Figure 3. A set of weight sliders to set parameters controlling the relative weights of different contributions.
+
+Additionally, parameters, cred, and grain are all scoped to a community. This means that the community that supports a project sets and controls the parameters. For that matter, it’s worth reiterating that (lowercase) cred and grain generated for a project are specific to that project. SourceCred is about providing community control, keeping humans in the loop, both for gaming mitigation and general adaptability. At the bottom line, it’s important to keep the humans in control, because good governance technology only works with the consent of the governed.
+
+#### Manual input:
+
+It’s also worth noting that not all contributions occur on platforms with automated data fetching, and thus the capability exists to individually add them to the weighted graph directly at this stage. It’s clear that there will always be data that has to be incorporated manually, and how to incorporate those contributions, especially those not easily quantified, remains an active area of discussion.
+
+#### Merging graphs:
+
+A plugin will use the API for the collaborative workspace app or platform to generate a graph of nodes and edges. One interesting caveat is that there isn't necessarily one most-intuitively correct way to merge two weighted graphs if they share either edges or nodes (i.e. how would we calculate the combined weight). Currently, the graph nodes are nearly entirely disjoint, the weights are added together, and identities are collapsed into one.
+
+#### Summary up to this point:
+
+So at this point, we’ve got data from all our sources combined in one weighted, directed graph, where contributions and contributors are nodes, interactions and connections between nodes are edges, each have a relative weight based on parameters the community has set, and we’re ready to run PageRank on it.
+
```

Comment:
each has?

Reply:
I definitely meant "both edges and nodes" instead of "each." 
I'll also be changing this bit to explicitly talk about converting from a weights graph to a Markov process graph to a Markov process, before calculating the stationary distribution, so there'll be a new section or two right after this, which might lead me to change this transition.